### PR TITLE
Fix pagedown issues

### LINKED ISSUE #117: Markdown Editor features MIA (this is both dev and prod)
No format buttons, no live preview, and it appears markdown is not being processed.  Markdown was functioning a few days ago.

screen shot 2016-04-23 at 12 34 22 pm

screen shot 2016-04-23 at 12 33 15 pm

### PR DESCRIPTION
**Pull request by risu-kun**
To fix #117:

- Got rid of turbolinks
- Added some jQuery to the posts#show page to convert the markdown to HTML in the browser
- Added white boxes around the post body
- Made the replies display under the posts instead of to the side (this makes it apparent that issue #100 is still alive, unfortunately :cry:)

Please pull this down, try to break it, and let me know what you think.

### CODE REVIEW

**Comment by kenmazaika** [comment]
Looks good to me.  Thoughts @amarkpark?

**Comment by dvanetten03** [comment]
Looks good to me, too.

**Comment by risu-kun** [comment]
I snuck in a small edit to fix issue #102 while I was at it.

File: app/views/posts/show.html.erb [hunk 1]
```diff
@@ -1,27 +1,45 @@
...
+  $(function() {
+    $('.wmd-output').each(function(i) {
+      var converter = new Markdown.Converter();
+      Markdown.Extra.init(converter);
+      var content = $.trim($(this).text());
+      $(this).html(converter.makeHtml(content));
+    });
+  });
+</script>
+
```
Comment (amarkpark) [inline app/views/posts/show.html.erb]: @risu-kun you should save the above script as a Gist for adding markdown functionality to views.

**Comment by amarkpark** [comment]
Really nice job @risu-kun

**Pull request merged**